ci-operator/config/openshift/console/openshift-console-master.yaml

```diff
@@ -90,6 +90,25 @@ tests:
         requests:
           cpu: 100m
     workflow: ipi-azure
+- artifact_dir: /tmp/artifacts
+  as: ceph-storage-plugin
+  steps:
+    cluster_profile: aws
+    test:
+    - as: test
+      cli: latest
+      commands: ./test-prow-e2e.sh ceph
+      env:
+      - default: m4.2xlarge
+        name: COMPUTE_NODE_TYPE
+      - default: "1"
+        name: ZONES_COUNT
+      from: pipeline:src
+      resources:
+        requests:
+          cpu: 100m
+          memory: 10Gi
```

Comment:
The job seems ok now, but I think that 10G is too much. Can we try to rehearse the job with fewer memory request? Maybe 5Gi?